components/constellation/webview.rs

```diff
@@ -17,6 +17,12 @@ pub struct WebViewManager<WebView> {
 
     /// Whether the latest webview in focus order is currently focused.
     is_focused: bool,
+
+    /// Webviews that are being shown by the embedder, regardless of whether they are actually visible.
+    shown_webviews: HashSet<TopLevelBrowsingContextId>,
+
+    /// Webviews that are known to be invisible, regardless of whether they are being shown.
+    invisible_webviews: HashSet<TopLevelBrowsingContextId>,
```

Comment:
The "shown" versus "visible" distinction is a bit hard to grasp! Is it something like "in the scene" versus "in the scene, but hidden?"

Reply:
clarified in 9947d44e0c0b5e41128725e0e8e6cf8919890b60